OAuthSwift/OAuthSwiftCredential.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 
-public class OAuthSwiftCredential {
+public class OAuthSwiftCredential: NSCoding {
```

Comment:
I don't think NSCoding has any dependencies on NSObjects. Looking at the Swift protocol declaration:

```swift
protocol NSCoding {
    
    func encodeWithCoder(aCoder: NSCoder)
    init(coder aDecoder: NSCoder) // NS_DESIGNATED_INITIALIZER
}
```

It doesn't seem to have any dependency on NSObject functionality.